toolkit/tools/pkg/imagecustomizerlib/cosicommon.go

```diff
@@ -188,5 +207,81 @@ func populateMetadata(data ImageBuildData) error {
 		return fmt.Errorf("failed to calculate sha384 of %s:\n%w", data.Source, err)
 	}
 	data.Metadata.Image.Sha384 = sha384
+
+	err = populateVerityMetadata(data.VeritySource, data.Metadata.Verity)
+	if err != nil {
+		return fmt.Errorf("failed to populate verity metadata: %w", err)
+	}
+	return nil
+}
+
+func populateVerityMetadata(source *string, verity *Verity) error {
```

Comment:
`populateMetadata` and `populateVerityMetadata` are both populating a `ImageFile` type. You should see if there is a way to deduplicate the code.